test_unstructured/partition/test_auto.py

```diff
@@ -342,8 +345,11 @@ def test_auto_partition_pdf_from_file(pass_file_filename, content_type):
     assert isinstance(elements[0], Title)
     assert elements[0].text.startswith("LayoutParser")
 
+    # NOTE(alan): Xfail since new model misses the first word Zejiang
+    request.applymarker(pytest.mark.xfail)
```

Comment:
Do we need both the `xfail` and the update to the test below? If we plan in addressing the bounding box issue in inference I think just the `xfail` make sense.

Reply:
Oops, I meant to keep the old one with "Zejiang". I must have missed this when merging.

Reply:
Restored Zejiang in [988a7dd](https://github.com/Unstructured-IO/unstructured/pull/662/commits/988a7dda6f5a4a14769b2e5496853209cf844b41).